dist/src/main/resources/rules/file-mappings.windup.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/v1/xml" id="XmlFileMappings">
+    <rules>
+        <rule>
```

Comment:
To give an example of what I mean, take a look at the rule report. Each rule from this file will be preceded by a rule that looks something like this:

```
    "addRule()
    withId("GeneratedID_XmlFileMappings_1")
```

This is because the Rule adds a blank rule before processing the file-mapping Rule element.